src/Page.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Counter from './Conter';
+import Buttons from './Buttons';
+
+function Page({ count, onClick }) {
+  return (
+    <div>
+      <p>Hello, Counter!</p>
+      <Counter count={count} onClick={(i) => onClick(i)} />
+      <Buttons onClick={(i) => onClick(i)} />
```

Comment:
```suggestion
      <Counter count={count} onClick={onClick} />
      <Buttons onClick={onClick} />
```
위의 suggestion 처럼 사용하게 되면 더 간략하게 사용할수 있을거 같아요 :)